POEApi.Model/Gem.cs

```diff
@@ -6,13 +6,41 @@ namespace POEApi.Model
     public class Gem : SocketableItem
     {
         public int Level { get; set; }
+        public double LevelProgress { get; private set; }
```

Comment:
Add a comment explaining what `LevelProgress` is.  If I remember correctly from the API, it's the fraction the gem has progressed to the next level? (`LevelExperienceProgress` might be a better, more explicit name, then.)